windows-terminal-quake/Toggler.cs

```diff
@@ -65,6 +65,9 @@ public Toggler(string[] args)
 			// Toggle on hotkey(s)
 			HotKeyManager.HotKeyPressed += (s, a) =>
 			{
+				// If the current focus'd window is ignored, return (do nothing)
+				if (FocusTracker.CheckFocusIgnoreHotKey()) return;
```

Comment:
If we can come to a more descriptive name for the setting, this method should be renamed to reflect said setting.

Reply:
easy peasy :)